src/environments/environment.common.ts

```diff
@@ -216,4 +216,5 @@ export const environment: GlobalConfig = {
   theme: {
     name: 'default',
   },
+  rewriteDownloadUrls: false,
```

Comment:
Tiny thing. Let's add a comment here to describe this new setting.  Something like:
`// Whether the UI should rewrite file download URLs to match its domain. Only necessary to enable when running UI and REST API on separate domains.`

Reply:
Done